Lesson23/Kadir/filter.js

```diff
@@ -353,6 +362,9 @@ const studentsExercise9 = [
 	},
 ];
 
+const studentsExercise9Result = studentsExercise9.filter((student) => student.grade>=('C-'));
```

Comment:
This solution would not work as letter comparison does not work in the same way as number comparison. See this article for more details: https://medium.com/@larry.sassainsworth/string-comparison-in-javascript-8f8aaed91bcb

The solution could be like the one below:


```suggestion
const studentsExercise9Result = studentsExercise9.filter((student) => student.grade.toUpperCase() !== ('C-') && student.grade[0].toUpperCase() !== ('D') && student.grade[0].toUpperCase() !== ('E');
```